File: C#/백준/Bronze/2975. Transactions/Transactions.cs
using System;

namespace q{
    class w{
        static void Main(String[] args){
            while(true){
                String[] aa = Console.ReadLine().Split();
                int a = int.Parse(aa[0]);
                String s = aa[1];
                int b = int.Parse(aa[2]);
                if (a==0 && s=="W" && b==0){
                    break;
                }
                if (s=="W"){
                    if (a-b<-200){
                        Console.WriteLine("Not allowed");
                    }
                    else{
                        Console.WriteLine(a-b);
                    }
                }
                else{
                    Console.WriteLine(a+b);
                }
            }
        }
    }
}

File: Python/백준/Bronze/2975. Transactions/Transactions.py
while(True):
    aa = input().split()
    a = int(aa[0]); b = int(aa[2]); s = aa[1]
    if a==0 and b==0 and s=='W':
        break
    if s=='W':
        a-=b
        if (a<-200):
            print('Not allowed')
        else:
            print(a)
    else:
        a+=b
        print(a)